### PR TITLE
Trimming the examples in the Async overview

### PR DESCRIPTION
**Pull request by matthewjohnston4**
I've removed any specifics from these examples. Important to note that these overviews are intended for absolute beginners, hence the dumbed-down English. We want to be less concerned with the technical details in these overviews as with selling the 'why use this' of each feature - that said, technical inaccuracies are obviously to be removed!

### CODE REVIEW

**Comment by matthewjohnston4** [comment]
cc @fredemmott and @jwatzman

File: guides/hack/00-overview/03-async.md [hunk 1]
```diff
@@ -67,19 +61,19 @@ When Hack encounters an `await`, it knows that it needs to stop and wait for an
...
+* Second `makeBlockingRequest()` call for `$y`
 * Await the completion of the `$x` call
 * Print out the result of that call, `$result_x`
 * `$y` call completes separately
 
-Here are the important things to note:
+Here are the main things to notice:
 
-* Both async calls for `$x` and `$y` can overlap and while the first is stuck on `curl_exec` the second might be executing other lines of code.
+* Both `makeBlockingRequest` calls can overlap and while the first is stuck on the I/O instruction, the second might be executing other lines of code - it is very important to note that this is **not** multithreading, please read the [Async Documentation](../async/introduction.md) where we explain this more.
```
Comment (fredemmott) [inline guides/hack/00-overview/03-async.md]: They can not overlap if they do anything blocking

**Comment by fredemmott** [comment]
conflicts with #61

**Comment by fredemmott** [comment]
Closing given #61 is merged. Leaving the branch up for a day or two.

**Pull request closed without merging**